### PR TITLE
New package: GAI.GAI version 1.0.0

### LINKED ISSUE #202501: [Package Request]: Generative AI Tools for Beginner
### How can we help?

I would like help so I can submit the manifest.

### Please read and ensure the following

- [x] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://webpath.iche2.com/app/gai/download_en.html
Publisher: Heshan Aisipai E-commerce Co., Ltd.
Package Name: Gai
Description: Generative AI Tools for Beginner
Package Version: 1.0.0
Installer Url: https://webpath.iche2.com/release/Gai-1.0.0-Setup.exe

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #202501

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

## Manual validation

> [!NOTE]
> 
> After installation, I can't launch the application from **Startup Panel**. Further investigation needed.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/202507)

### CODE REVIEW

**Comment by Dragon1573** [comment]
## Manual validation

Verified in _Windows Sandbox_ instance. Further investigation needed.

```text
--> Installing WinGet
--> Disabling safety warning when running installers
Tip: you can type 'Update-EnvironmentVariables' to update your environment variables, such as after installing a new software.

--> Configuring Winget
已启用管理员设置 'LocalManifestFiles'。
已启用管理员设置 'LocalArchiveMalwareScanOverride'。

--> Installing the Manifest 1.0.0

已找到 Gai [GAI.GAI] 版本 1.0.0
此应用程序由其所有者授权给你。
Microsoft 对第三方程序包概不负责，也不向第三方程序包授予任何许可证。
正在下载 https://webpath.iche2.com/release/Gai-1.0.0-Setup.exe
  ██████████████████████████████  26.6 MB / 26.6 MB
已成功验证安装程序哈希
正在启动程序包安装...
已成功安装

--> Refreshing environment variables

--> Comparing ARP Entries

DisplayName       DisplayVersion Publisher                           ProductCode                                Scope
-----------       -------------- ---------                           -----------                                -----
Gai version 1.0.0 1.0.0          Heshan Aisipai E-commerce Co., Ltd. {84231572-4192-4BA0-BB22-C239D1B8ABFB}_is1 Machine
```

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> 2024-12-27 09:54:05.658 [CLI ] Package hash verification failed. SHA256 in manifest [7be0ad53eda3e6a08f70843eb068b1c6fba880c7a33672a98192045869c1e856] does not match download [7b2ad08a6ef5b926407cea8147c4c80c0c32718c17378b24ca1c779a6d335f13] with file size [28754473] and content type []

(Automated response - build 931.)

File: manifests/g/GAI/GAI/1.0.0/GAI.GAI.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+AppsAndFeaturesEntries:
+- Publisher: Heshan Aisipai E-commerce Co., Ltd.
+  ProductCode: '{84231572-4192-4BA0-BB22-C239D1B8ABFB}_is1'
+ElevationRequirement: elevatesSelf
+InstallationMetadata:
+  DefaultInstallLocation: '%ProgramFiles%\Gai'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://webpath.iche2.com/release/Gai-1.0.0-Setup.exe
+  InstallerSha256: 7BE0AD53EDA3E6A08F70843EB068B1C6FBA880C7A33672A98192045869C1E856
```
Comment (stephengillie) [inline manifests/g/GAI/GAI/1.0.0/GAI.GAI.installer.yaml:25]:
```suggestion
  InstallerSha256: 7B2AD08A6EF5B926407CEA8147C4C80C0C32718C17378B24CA1C779A6D335F13
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
> Manual Validation ended with:
> 
> > 2024-12-27 09:54:05.658 [CLI ] Package hash verification failed. SHA256 in manifest [7be0ad53eda3e6a08f70843eb068b1c6fba880c7a33672a98192045869c1e856] does not match download [7b2ad08a6ef5b926407cea8147c4c80c0c32718c17378b24ca1c779a6d335f13] with file size [28754473] and content type []
> 
> (Automated response - build 931.)

File: manifests/g/GAI/GAI/1.0.0/GAI.GAI.installer.yaml (hunk 1 above)
Comment (huanguan1978) [reply manifests/g/GAI/GAI/1.0.0/GAI.GAI.installer.yaml:25]: My download page has a SHA256 checksum of 7B2AD08A6EF5B926407CEA8147C4C80C0C32718C17378B24CA1C779A6D335F13.

**Comment by huanguan1978** [comment]
@stephengillie @Dragon1573 
I don't understand the recent messages: "Unresolved conversations," "Some checks were not successful," and "Merging is blocked."  What action do I need to take?

**Comment by Dragon1573** [comment]
> I don't understand the recent messages: "Unresolved conversations," "Some checks were not successful," and "Merging is blocked."

Those are information related to me. Don't care if you're not the pull request author.

> What action do I need to take?

Nope. I'm the author and disabled **"Allow edits and access to secrets by maintainers"** switch. So all modifications can only done by myself.

**Comment by Dragon1573** [comment]
> During installation testing, this application failed to install without user input.
> 
> Please investigate a fix and resubmit the Pull Request.
> 
> Template: msftbot/validationError/installation/unattended

CC @stephengillie 🙏🏼

**Comment by huanguan1978** [comment]
I saw a recent message reading "Validation-Unattended-Failed".  I tested running "Gai-1.0.0-Setup.exe /vervslient" from the command line, and it runs, but a prompt appears: "Do you want to allow this app from an unknown publisher to make changes to your device".  This prompt requires manual confirmation before installation.  Is this the cause?

**Comment by stephengillie** [comment]
Verify:
image
I'm not sure why it wasn't able to detect my connection.
image

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
> I saw a recent message reading "Validation-Unattended-Failed". I tested running "Gai-1.0.0-Setup.exe /vervslient" from the command line, and it runs, but a prompt appears: "Do you want to allow this app from an unknown publisher to make changes to your device". This prompt requires manual confirmation before installation. Is this the cause?

This prompt is bypassed in some way in the pipeline. In my VMs, I click through it. So this shouldn't be the cause of the error. 

Unfortunately, since a recent pipeline change, run logs are no longer emitted. So the actual cause is hidden from public view. In lieu of these, I've scripted a local manifest install and Defender scan.

**Comment by huanguan1978** [comment]
@stephengillie Based on your screenshot and feedback "I'm not sure why it wasn't able to detect my connection," I can conclude that the network in your virtual machine is unable to connect to external networks, such as www.google.com, www.apple.com. 

 If the Winget community repository does not require the application to be offline, the application should be able to display its start page and allow you to choose whether to run it. This confirms the installation portion is complete.

What next steps do you need from me as the developer?

**Comment by huanguan1978** [comment]
C:\Users\14714>winget search GAI.GAI
名称 ID      版本  源
--------------------------
Gai  GAI.GAI 1.0.0 winget

C:\Users\14714>winget show GAI.GAI
已找到 Gai [GAI.GAI]
版本: 1.0.0
发布者: Heshan Aisipai E-commerce Co., Ltd.
发布服务器 URL: https://www.iche2.com/enterprise/
描述:
  [GAI] 是一款智能AI助手,为你提供高效便捷的创意内容生成体验。

==========================

Based on the above information, the winget community repository appears to have been updated successfully.

I want to update related files, such as `manifests/g/GAI/GAI/1.0.0/GAI.GAI.locale.zh-CN.yaml`.  How can I do that?  Should I fork https://github.com/microsoft/winget-pkgs, modify it, commit the changes, and push them?

**Comment by stephengillie** [comment]
Hi @huanguan1978,

> @ stephengillie Based on your screenshot and feedback "I'm not sure why it wasn't able to detect my connection," I can conclude that the network in your virtual machine is unable to connect to external networks, such as [www.google.com](http://www.google.com), [www.apple.com](http://www.apple.com).

But the VM can access these sites. Or at least, it should be able to. That's how it downloaded the package. And since this has only happened in a VM, it might be unique to my VMs. Which is why I approved the PR. 

> I want to update related files, such as manifests/g/GAI/GAI/1.0.0/GAI.GAI.locale.zh-CN.yaml. How can I do that? Should I fork https://github.com/microsoft/winget-pkgs, modify it, commit the changes, and push them?

Yes, exactly. It will go through a shorter review process, and then become available in the same way as the full manifest.

**Pull request merged**